### PR TITLE
Main to fiber

### PR DESCRIPTION
**Pull request by ValdisThomann**

### CODE REVIEW

File: src/utils/keyboard-rendering.ts [hunk 1]
```diff
@@ -453,10 +452,10 @@ export const getLabel = (
   // Full name
   let tooltipLabel: string = '';
   if (
-    isUserKeycodeByte(keycodeByte, basicKeyToByte) &&
+    isCustomKeycodeByte(keycodeByte, basicKeyToByte) &&
     selectedDefinition?.customKeycodes
   ) {
-    const userKeycodeIdx = getUserKeycodeIndex(keycodeByte, basicKeyToByte);
+    const userKeycodeIdx = getCustomKeycodeIndex(keycodeByte, basicKeyToByte);
```
Comment (wilba) [inline src/utils/keyboard-rendering.ts:458]: change userKeycodeIdx to customKeycodeIdx

File: src/utils/key.ts [hunk 2]
```diff
@@ -294,6 +307,385 @@ export function getShortNameForKeycode(keycode: IKeycode, size = 100) {
   return name;
 }
 
+export function mapEvtToKeycode(evt: KeyboardEvent) {
```
Comment (wilba) [inline src/utils/key.ts:310]: Was this function deleted in fiber branch and being added back in from main?

**Review by wilba — CHANGES_REQUESTED** [review]

File: src/utils/keyboard-rendering.ts (hunk 1 above)
Comment (ValdisThomann) [reply src/utils/keyboard-rendering.ts:458]: done

File: src/utils/advanced-keys.ts [hunk 3]
```diff
@@ -208,7 +216,14 @@ const topLevelModToString = (
   basicKeyToByte: Record<string, number>,
   byteToKey: Record<number, string>,
 ): string => {
+  debugger;
```
[the remark this answers is not in the document]
Comment (ValdisThomann) [reply src/utils/advanced-keys.ts:219]: done

File: src/utils/key.ts [hunk 4]
```diff
@@ -546,8 +559,8 @@ export function getKeycodes(): IKeycodeMenu[] {
           keys: 'num_0',
           title: 'Numpad 0',
         },
-        {name: '÷', code: 'KC_PSLS', keys: 'num_divide', title: 'Numpad ÷'},
-        {name: '×', code: 'KC_PAST', keys: 'num_multiply', title: 'Numpad ×'},
+        {name: '/', code: 'KC_PSLS', keys: 'num_divide', title: 'Numpad /'},
```
Comment (wilba) [inline src/utils/key.ts:562]: keep ÷ and ×

Comment (ValdisThomann) [reply src/utils/key.ts:562]: Ok. This was auto-merged though, which means they were removed in fiber for some reason.

Comment (ValdisThomann) [reply src/utils/key.ts:562]: Just confirming: you want ÷ and × instead of / and * ? Isn't the latter standard on a numpad?

Comment (wilba) [reply src/utils/key.ts:562]: yes

Comment (ValdisThomann) [reply src/utils/key.ts:562]: done

**Pull request merged**